packages/terra-aggregate-translations/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Removed
+  * Removed intl v3 and v4 support.
```

Comment:
Can we add an upgrade guide for this MVB?

Reply:
Yes, I think we should always accompany any breaking change with the relevant upgrade guide for our users.

Reply:
added [here](https://github.com/cerner/terra-toolkit/pull/739/files#diff-6cf94aac50699d6fc85ff446edc1a57df6e4384bf7b27ce1d9697887f45e1755)